test/helpers/ava.setup.js

```diff
@@ -0,0 +1,15 @@
+if (process.env.TEST === 'unit') {
```

Comment:
I'd prefer to keep ava.setup.js as-is for now. It also requires browser-env which I think was needed for some tests. Having this file in helpers may be a good place for it though.

Reply:
i will revert the file to its original setup